etc/bento.env

```diff
@@ -440,3 +440,19 @@ BENTO_CBIOPORTAL_SESSION_DATABASE_IMAGE_VERSION=6.0.15
 BENTO_CBIOPORTAL_SESSION_DATABASE_CONTAINER_NAME=${BENTOV2_PREFIX}-cbioportal-session-db
 BENTO_CBIOPORTAL_SESSION_DATABASE_DATA_DIR=${BENTO_FAST_DATA_DIR}/cbioportal/session_db
 # Uses BENTO_CBIOPORTAL_SESSION_NETWORK as the network
+
+# Monitoring
+BENTO_MONITORING_NETWORK=${BENTOV2_PREFIX}-monitoring-net
+BENTO_PRIVATE_GRAFANA_URL=${BENTOV2_PORTAL_PUBLIC_URL}/api/grafana
+BENTO_PUBLIC_GRAFANA_URL=${BENTOV2_PORTAL_PUBLIC_URL}/grafana
+BENTO_LOKI_IMAGE=grafana/loki
+BENTO_LOKI_IMAGE_VERSION=3.1.1
+BENTO_LOKI_CONTAINER_NAME=${BENTOV2_PREFIX}-loki
+BENTO_LOKI_PROD_TEMP_DIR=${BENTO_SLOW_DATA_DIR}/loki/tmp
```

Comment:
I'd simplify this to be just called `BENTO_LOKI_PROD_TEMP_DIR` if there isn't a different location for dev